dandelion-voting/test/voting.js

```diff
@@ -210,11 +221,12 @@ contract('Voting App', ([root, holder1, holder2, holder20, holder29, holder51, n
                 })
 
                 it('has correct state', async () => {
-                    const [isOpen, isExecuted, startBlock, snapshotBlock, supportRequired, minQuorum, y, n, execScript] = await voting.getVote(voteId)
+                    const [isOpen, isExecuted, startBlock, executionBlock, snapshotBlock, supportRequired, minQuorum, y, n, execScript] = await voting.getVote(voteId)
 
                     assert.isTrue(isOpen, 'vote should be open')
                     assert.isFalse(isExecuted, 'vote should not be executed')
                     assert.equal(startBlock.toString(), await getBlockNumber(), 'start block should be correct')
+                    assert.equal(executionBlock.toString(), startBlock.toNumber() + executionDelayBlocks + voteDurationBlocks)
```

Comment:
Consider also adding a fail message

Reply:
> Also what do you think about adding a test which ensures changing the execution delay doesn't affect already created votes?

It's a good point but I don't think it's necessary in this case as `changeExecutionDelayBlocks()` doesn't modify anything besides the `executionDelayBlocks` variable. And I trust that any future developer that modifies `changeExecutionDelayBlocks()` or the way in which a vote is stored will add such a test if it modifies the `executionDelay` of a vote. However, if you'd still like me to add it, I gladly will.